docs/Setup.md

```diff
@@ -1494,10 +1494,10 @@ $(function () {
 -   `_txt` **any** The graph definition
 -   `cb` **any** Callback which is called after rendering is finished with the svg code as inparam.
 -   `container` **any** Selector to element in which a div with the graph temporarily will be
-      inserted. In one is provided a hidden div will be inserted in the body of the page instead. The
-      element will be removed when rendering is completed.
+    inserted. In one is provided a hidden div will be inserted in the body of the page instead. The
```

Comment:
Should be "If one is provided..." instead of "In one is provided..."

Reply:
Thanks @lorint.